andela/travel_tool_front
# #161690447 Render a conditional loader component

#### What does this PR do?
Renders the loader component until the appropriate data for the selected guest house is fetched

#### Description of Task to be completed?
- In the guest house details page, have a conditional statement to render the loader component when data is still being fetched from the server.

#### How should this be manually tested?
* Clone the repos: ``` https://github.com/andela/travel_tool_front ``` and ``` https://github.com/andela/travel_tool_back ```
* Checkout to the branch ``` bg-fix-data-display-in-guest-house-details-161690447 ``` on the ``` travel_tool_front ```.
* Start up the back-end and front-end with ``` yarn start:dev ``` and ``` yarn start ``` respectively.
* Login to the app on the front end, and ensure you have the travel admin privilege.
* Navigate to the Residence Manage page, and click on a guest-house
* You should see a loading spinner while the call is still being made to the server.

#### Any background context you want to provide?
* N/A

#### What are the relevant pivotal tracker stories?
[161690447](https://www.pivotaltracker.com/story/show/161690447)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-11-14 at 2 00 27 pm" src="https://user-images.githubusercontent.com/29521319/48484884-a1213500-e817-11e8-8883-faafeb527ddc.png">
<img width="1440" alt="screen shot 2018-11-14 at 2 00 31 pm" src="https://user-images.githubusercontent.com/29521319/48484902-ab433380-e817-11e8-866e-2b0b0332304b.png">


#### Questions:
